manim/animation/specialized.py

```diff
@@ -1,85 +0,0 @@
-"""Animations for highly specialiced applications."""
-
-__all__ = ["MoveCar", "Broadcast"]
-
-
-import operator as op
-
-from ..animation.composition import LaggedStart
-from ..animation.transform import ApplyMethod
-from ..animation.transform import Restore
-from ..mobject.geometry import Circle
-from ..mobject.svg.drawings import Car
-from ..mobject.types.vectorized_mobject import VGroup
-from ..utils.config_ops import digest_config
-from ..utils.space_ops import get_norm
-from ..utils.color import BLACK, WHITE
-
-
-class MoveCar(ApplyMethod):
-    CONFIG = {
-        "moving_forward": True,
-        "run_time": 5,
-    }
-
-    def __init__(self, car, target_point, **kwargs):
-        self.check_if_input_is_car(car)
-        self.target_point = target_point
-        super().__init__(car.move_to, target_point, **kwargs)
-
-    def check_if_input_is_car(self, car):
-        if not isinstance(car, Car):
-            raise TypeError("MoveCar must take in Car object")
-
-    def begin(self):
-        super().begin()
-        car = self.mobject
-        distance = get_norm(
-            op.sub(
-                self.target_mobject.get_right(),
-                self.starting_mobject.get_right(),
-            )
-        )
-        if not self.moving_forward:
-            distance *= -1
-        tire_radius = car.get_tires()[0].get_width() / 2
-        self.total_tire_radians = -distance / tire_radius
-
-    def interpolate_mobject(self, alpha):
-        ApplyMethod.interpolate_mobject(self, alpha)
-        if alpha == 0:
-            return
-        radians = alpha * self.total_tire_radians
-        for tire in self.mobject.get_tires():
-            tire.rotate_in_place(radians)
-
-
-class Broadcast(LaggedStart):
```

Comment:
I can see the need to remove MoveCar, but Broadcast seems like it might be of general use. Is there an example using Broadcast somewhere?